Readme.md

```diff
@@ -82,13 +82,32 @@ _Tip:_ Add the `*.generated.swift` pattern to your `.gitignore` file to prevent
 ### Manually
 
 0. Add the [R.swift.Library](https://github.com/mac-cain13/R.swift.Library#Installation) to your project
-1. [Download](https://github.com/mac-cain13/R.swift/releases) a R.swift release, unzip it and put it into your source root directory
+1. [Download](https://github.com/mac-cain13/R.swift/releases) a R.swift release, unzip it and put it into your source root directory OR install the binary via homebrew. For more detail check [Homebrew](#homebrew)
```

Comment:
The installation instruction in step 3 is using `"$SRCROOT/rswift"`, that won't work when you use Homebrew.